conmon-rs/server/src/server.rs

```diff
@@ -222,6 +222,8 @@ impl Server {
         }
     }
 
+    const SYSTEMD_CGROUP_ARG: &'static str = "--systemd-cgroup";
```

Comment:
we I think should actually toggle this based on the passed cgroup manager I think. Maybe a flag at startup, as we'll use one cgroup manager per pod (more acurately per node for cri-o, but possibly per pod for podman)

Reply:
Yep, added it via a configuration variable.